src/globus_sdk/services/compute/client.py

```diff
@@ -101,18 +101,22 @@ def get_endpoint_status(self, endpoint_id: UUIDLike) -> GlobusHTTPResponse:
         """  # noqa: E501
         return self.get(f"/v2/endpoints/{endpoint_id}/status")
 
-    def get_endpoints(self) -> GlobusHTTPResponse:
+    def get_endpoints(self, role: str | None = None) -> GlobusHTTPResponse:
```

Comment:
As this is a new parameter, I recommend switching from `None` as the sentinel value to `globus_sdk.MISSING`, which has the type `globus_sdk.MissingType`:

```suggestion
    def get_endpoints(self, role: str | MissingType = MISSING) -> GlobusHTTPResponse:
```

We're making a major effort in the v4 branch to convert to a well-known parameter sentinel value so that there's no ambiguity. Since there's no legacy concern with this new parameter, using `MISSING` from the get-go will make this merge easily into the v4 branch.

Reply:
Thanks for the super quick review. I have these changes merged.